examples/EvaluationDataToApplicationInsights/Pages/Shared/_Layout.cshtml

```diff
@@ -0,0 +1,95 @@
+﻿<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>@ViewData["Title"] - EvaluationDataToApplicationInsights</title>
+    <link rel="stylesheet" href="~/lib/bootstrap/dist/css/bootstrap.min.css" />
+    <link rel="stylesheet" href="~/css/site.css" asp-append-version="true" />
+    <link rel="stylesheet" href="~/EvaluationDataToApplicationInsights.styles.css" asp-append-version="true" />
+    @Html.Raw(JavaScriptSnippet.FullScript)
+
+    <script src="~/lib/jquery/dist/jquery.min.js"></script>
+    <script src="~/lib/bootstrap/dist/js/bootstrap.bundle.min.js"></script>
+    <script src="~/js/site.js" asp-append-version="true"></script>
+    <script>
+        function getCookie(cname) {
+            let name = cname + "=";
+            let decodedCookie = decodeURIComponent(document.cookie);
+            let ca = decodedCookie.split(';');
+            for (let i = 0; i < ca.length; i++) {
+                let c = ca[i];
+                while (c.charAt(0) == ' ') {
+                    c = c.substring(1);
+                }
+                if (c.indexOf(name) == 0) {
+                    return c.substring(name.length, c.length);
+                }
+            }
+            return "";
+        }
+
+        function deleteCookie(name, path, domain) {
+            if (getCookie(name)) {
+                document.cookie = name + "=" +
+                    ((path) ? ";path=" + path : "") +
+                    ((domain) ? ";domain=" + domain : "") +
+                    ";expires=Thu, 01 Jan 1970 00:00:01 GMT";
+            }
+        }
+
+        appInsights.setAuthenticatedUserContext(getCookie("username"));
+    </script>
+</head>
+<body>
+    <header>
+        <nav class="navbar navbar-expand-sm navbar-toggleable-sm navbar-light bg-white border-bottom box-shadow mb-3">
+            <div class="container">
+                <a class="navbar-brand" asp-area="" asp-page="/Index">EvaluationDataToApplicationInsights</a>
+                <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target=".navbar-collapse" aria-controls="navbarSupportedContent"
+                        aria-expanded="false" aria-label="Toggle navigation">
+                    <span class="navbar-toggler-icon"></span>
+                </button>
+                <div class="navbar-collapse collapse d-sm-inline-flex justify-content-between">
+                    <ul class="navbar-nav flex-grow-1">
+                        <li class="nav-item">
+                            <a class="nav-link text-dark" asp-area="" asp-page="/Index">Home</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link text-dark" asp-area="" asp-page="/Checkout">Checkout</a>
+                        </li>
+                        <feature name="FeatureA">
```

Comment:
I think it would be better to move this into Index.cshtml
Something like:
<feature name="FeatureA">
    <p>Since you are seeing this, that means the "FeatureA" is enabled for the current user because he/she falls within the default rollout percentage of the targeting filter.</p>
</feature>
In this way, it would be easier for user to understand how this example project works.

Reply:
We can do that. Currently they're in _Layout so they show on the top for all pages (Index, Checkout, and Error). Moving it to Index means you'll only see it on that page. Adding pages for each feature feels like too much for this example.

The main flow of this example is to Checkout. So I definitely want it present on that page. Would you prefer I add your suggestion ^ to the checkout page, add your example to only index, or leave it in the shared header?

Reply:
Sounds good!